files.js

```diff
@@ -41,21 +57,33 @@ function loadTypedArrayFile(filename, Type, cb) {
     .catch(cb)
 }
 
-function saveBitsetFile(filename, seq, bitset, cb) {
+function saveBitsetFile(filename, version, seq, bitset, cb) {
   bitset.trim()
-  saveTypedArrayFile(filename, seq, bitset.count, bitset.words, cb)
+  saveTypedArrayFile(
+    filename,
+    seq,
+    version,
+    bitset.count,
+    bitset.size(),
+    bitset.words,
+    cb
+  )
 }
 
 function loadBitsetFile(filename, cb) {
-  loadTypedArrayFile(filename, Uint32Array, (err, { tarr, count, seq }) => {
-    if (err) cb(err)
-    else {
-      const bitset = new TypedFastBitSet()
-      bitset.words = tarr
-      bitset.count = count
-      cb(null, { seq, bitset })
+  loadTypedArrayFile(
+    filename,
+    Uint32Array,
+    (err, { version, seq, count, tarr }) => {
```

Comment:
you are missing bitsetSize here ;)

Reply:
It's not necessary, right? bitsetSize is only relevant in `loadBitsetFile`, and actually it would only be necessary once we start using that field, for that idea #8